src/components/TextInput/BaseTextInput/index.tsx

```diff
@@ -77,72 +109,72 @@ function BaseTextInput(props) {
     );
 
     const activateLabel = useCallback(() => {
-        const value = props.value || '';
+        const newValue = value ?? '';
```

Comment:
```suggestion
        const newValue = value || '';
```

Since both are strings, maybe is safer to keep using `||`. WDYT?